polislite.py

```diff
@@ -1,21 +1,31 @@
+import yaml
 import numpy as np
 from sklearn.decomposition import PCA
 from scipy.cluster import hierarchy
 from sklearn.metrics import silhouette_score
 from collections import defaultdict
+from jinja2 import Template
+from pathlib import Path
 
 class PolisClusterer:
     def __init__(self, min_clusters=2, max_clusters=6):
         self.pca = PCA(n_components=2)
         self.min_clusters = min_clusters
         self.max_clusters = max_clusters
+        template_path = Path(__file__).parent / 'report_template.j2'
+        self.template = Template(template_path.read_text())
         
+    @staticmethod
+    def load_from_yaml(filepath):
+        with open(filepath) as f:
+            data = yaml.safe_load(f)
+        vote_map = {'agree': 1, 'disagree': -1}
+        votes = [[vote_map.get(v, 0) for v in user_votes] 
```

Comment:
```suggestion
        vote_map = {'agree': 1, 'disagree': -1, 'pass': 0}
        votes = [[vote_map.get(v) for v in user_votes]
```

To be more explicit about when someone unexpected happens, and avoid dividing assumptions between two lines :)